src/lib/tabs/tab-group.spec.ts

```diff
@@ -136,6 +139,53 @@ describe('MdTabGroup', () => {
       expect(component.handleSelection).toHaveBeenCalledTimes(1);
       expect(component.selectEvent.index).toBe(2);
     }));
+
+    it('should update tab positions and attach content when selected', fakeAsync(() => {
+      fixture.detectChanges();
+      let tabComponent = fixture.debugElement.query(By.css('md-tab-group')).componentInstance;
```

Comment:
Thanks, I tend to just default to `let` rather than `const`. Gotta break that habit.